### PR TITLE
Add Tizen studio 5.1 release notes

### PR DESCRIPTION
**Pull request by vivek-ellur**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/tizen-studio/release-notes/5-1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## IDE and tools
+
+### New features
+
+- Visual Studio (Windows)
+  - Added .NET Hot Reload support for Tizen dotnet projects using which code changes can be applied to a running app without restarting the app and without losing app state.
+  - Added support for NUIGadget template which is built as dotnet project and packaged as RPK. 
+### Fixed bugs
+  - Updated the Visual Studio path in the CLI to use Visual Studio 2022 for dotnet projects.
```
Comment (safir-srbd) [inline docs/application/tizen-studio/release-notes/5-1-release-notes.md:13]: Spacing issue.
```suggestion
  - Added support for NUIGadget template which is built as dotnet project and packaged as RPK. 
  
### Fixed bugs

  - Updated the Visual Studio path in the CLI to use Visual Studio 2022 for dotnet projects.

```

File: docs/application/tizen-studio/release-notes/5-1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Tizen Studio 5.1 Release Notes
+
+- Release Date: Mar 31, 2023
+
+## IDE and tools
+
+### New features
+
+- Visual Studio (Windows)
+  - Added .NET Hot Reload support for Tizen dotnet projects using which code changes can be applied to a running app without restarting the app and without losing app state.
```
Comment (safir-srbd) [inline docs/application/tizen-studio/release-notes/5-1-release-notes.md:10]:
```suggestion
  - Added .NET Hot Reload support for Tizen dotnet projects, using which code changes can be applied to a running app without restarting the app and without losing the app state.

```

File: docs/application/tizen-studio/release-notes/5-1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- Release Date: Mar 31, 2023
+
+## IDE and tools
+
+### New features
+
+- Visual Studio (Windows)
+  - Added .NET Hot Reload support for Tizen dotnet projects using which code changes can be applied to a running app without restarting the app and without losing app state.
+  - Added support for NUIGadget template which is built as dotnet project and packaged as RPK.
```
Comment (safir-srbd) [inline docs/application/tizen-studio/release-notes/5-1-release-notes.md:11]:
```suggestion
  - Added support for NUIGadget template which is built as a dotnet project and packaged as RPK. 

```

**Review by safir-srbd — COMMENTED** [review]
@vivek-ellur Hello. Hope you are well. Thank you for reopening this PR.

I have gone through the release note and have a few suggestions for you to implement. Please check.

**Review by sangwook96 — APPROVED** [review]
Great , Thank you very much ,I approved it now

**Comment by sangwook96** [comment]
@safir-srbd Hi, Could you review it again ?

**Comment by safir-srbd** [comment]
> @safir-srbd Hi, Could you review it again ?

Yes sure. We will go through another check and see if everything is in order.

**Review by safir-srbd — APPROVED** [review]
@vivek-ellur Thank you for implementing the suggested changes. Everything seems to be in order right now.

**Pull request merged**